### ISSUE TITLE
Updating grunge-stack to lastest versions

### ISSUE THREAD

**Issue by jarimustonen** [remix-run/grunge-stack#185]
### Have you experienced this bug with the latest version of the template?

Yes

### Steps to Reproduce

As the software evolves, the stack should be such that is could be updated to latest version. To this end, I suggest that grunge-stack is updated to latest versions.

I have tried this manually and was able to fix a few errors:

1. A few packages must be added explicitly (`@smithy/node-http-handler`, `@aws-sdk/lib-dynamodb`, and `@aws-sdk/client-dynamodb`)
2. `isbot` should be retained at the version `3.8.0` due to breaking changes in the version `4.0.0`
3. `remix.config.js` must be added the following configuration: `browserNodeBuiltinsPolyfill: { modules: { querystring: true } }`

Finally there is also an error related to cypress tests. The tests fail with:

```
Lambda init error: The requested module 'msw' does not provide an export named 'rest'
http://localhost:8811 timed out on retry 9 of 3, elapsed 95619ms, limit 90000ms
Error: Response code 500 (Internal Server Error)
```

I'm at lost here and really do not know what to do with this. If someone can help with this one, I could make pull request. Or maybe the Remix team would like to update the stack..

### Expected Behavior

N/A

### Actual Behavior

N/A

**Comment by McNouvion** [comment]
I also ran in the cypress issue when it ran via the github action .... I got around it by updating `mocks/index.js` and replacing 'rest' with 'http'.

```
import { http } from "msw";
...  
http.post(`...
```

For a short time I had some weird behavior where this change was not working locally but successful in Github ... Eventually it did start working locally as well but I don't remember if/what I did to fix it.

**Comment by gustavorino** [comment]
Any chance this project will be updated to use the vite version? is there anyone working on this?

**Comment by quantuminformation** [comment]
its a template, you are free to upgrade it yourself and sumbit a PR

**Comment by gustavorino** [comment]
> its a template, you are free to upgrade it yourself and sumbit a PR

This is exactly why I asked, I am considering updating it, but won't do it if, lets say, someone else is already working on it.

**Comment by quantuminformation** [comment]
I think everyone is happy paying the bills with esbuild

but esbuild is becoming a drag so I'm getting closer to trying vite again by myself

**Comment by davisg123** [comment]
@gustavorino I have the build working here if you want to help https://github.com/davisg123/grunge-stack/tree/vite

I've been unable to figure out how to run the architect sandbox along with vite:dev. There is some discussion about it https://github.com/remix-run/remix/discussions/8836 but I haven't figured out what the run dev command needs to be

**Comment by MichaelDeBoey** [comment]
Closed by #186
